geomesa-hbase/geomesa-hbase-tools/src/main/scala/org/locationtech/geomesa/hbase/tools/export/HBaseBinExportCommand.scala

```diff
@@ -11,11 +11,14 @@ package org.locationtech.geomesa.hbase.tools.export
 import com.beust.jcommander.Parameters
 import org.locationtech.geomesa.hbase.data.HBaseDataStore
 import org.locationtech.geomesa.hbase.tools.HBaseDataStoreCommand
+import org.locationtech.geomesa.tools.export.BinExportCommand.BinExportParams
+import org.locationtech.geomesa.tools.export.{BinExportCommand, ExportCommand}
 import org.locationtech.geomesa.tools.{CatalogParam, RequiredTypeNameParam}
-import org.locationtech.geomesa.tools.export.{BinExportCommand, BinExportParams}
 
+@deprecated("HBaseExportCommand")
```

Comment:
use the regular 'HBaseExportCommand' - that's what i was trying to convey in the deprecated string, obv failed :)